### PR TITLE
Track lines polygons

### PR DESCRIPTION
**Pull request by lucienicolier**
I change tracking to tracking session

### CODE REVIEW

File: en/user-guide/track_lines_polygons.rst [hunk 1]
```diff
@@ -1,12 +1,12 @@
 .. _track_linges_polygons:
 
-Tracking of lines and polygons
+Tracking session of lines and polygons
```
Comment (signedav) [inline en/user-guide/track_lines_polygons.rst]:
```suggestion
Tracking of lines and polygons
```

File: en/user-guide/track_lines_polygons.rst [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 ==============================
 
-You have the possibility to do tracking on lines and polygons.
+You have the possibility to do tracking session on lines and polygons.
 
 It's available when GPS is enabled. 
-Tracking can be started over the legend - long press on the layer.
+Tracking session can be started over the legend - long press on the layer.
```
Comment (signedav) [inline en/user-guide/track_lines_polygons.rst]:
```suggestion
A tracking session can be started over the legend - long press on the layer.
```

File: en/user-guide/track_lines_polygons.rst [hunk 3]
```diff
@@ -1,12 +1,12 @@
 .. _track_linges_polygons:
 
-Tracking of lines and polygons
+Tracking session of lines and polygons
 ==============================
 
-You have the possibility to do tracking on lines and polygons.
+You have the possibility to do tracking session on lines and polygons.
```
Comment (signedav) [inline en/user-guide/track_lines_polygons.rst]:
```suggestion
You have the possibility to do tracking on lines and polygons.
```

**Comment by lucienicolier** [comment]
@signedav is it ok to merge?

**Pull request merged**